ui/desktop/app/templates/scopes/scope/projects/settings.hbs

```diff
@@ -13,7 +13,7 @@
     >
       {{t 'titles.settings'}}
       {{! TODO: Add a link to the settings documentation }}
-      {{! <DocLink @doc='setting' @iconSize='large' /> }}
+      {{! <DocLink @doc='setting' @iconSize='24' /> }}
```

Comment:
I think this TODO can be removed since i don't think it ended up being necessary? cc @DhariniJeeva

Reply:
oh yeah good call, we can remove this